src/zkevm_specs/evm/table.py

```diff
@@ -212,6 +218,27 @@ class CallContextFieldTag(IntEnum):
     StateWriteCounter = auto()
 
 
+class TxLogFieldTag(IntEnum):
+    """
+    Tag for RWTable lookup with tag TxLog, which is used to index specific
+    field of TxLog.
+    """
+
+    # The following are write-only data inside a transaction, they will be written in
+    # State circuit directly.
+    Address = auto()  # address of the contract that generated the event
+    Topics = auto()  # list of topics provided by the contract
+    Data = auto()  # log data in bytes
+    BlockNumber = auto()  # block number which logs are belong to
+    TxHash = auto()  # hash of the transaction
+    TxIndex = auto()  # index of the transaction in the block
+    # The Removed field is true if this log was reverted due to a chain reorganisation.
+    # it is intermediate value, hence not count for state.
+    # Removed = auto()
```

Comment:
Do we mean to remove `Removed`?

Reply:
`Removed` field is an Intermediate variable to event subscriber, which only be true during chain reorganisation.  you can refer to https://github.com/ethereum/go-ethereum/blob/master/core/types/log.go#L53L55.  and blog https://ethereum.stackexchange.com/questions/77871/web3-ethereum-log-subscriptions-removed-field
in other words, it will always be false when it is in the block time. so by default I think we don't need to care about it as we focus on the final block.  does it make sense ?

Reply:
That makes sense. Since it's always false inside the circuit, we are safe to remove `Removed`?
```suggestion
```

Reply:
yeah, I will remove it , thanks

Reply:
removed in [0912af8](https://github.com/appliedzkp/zkevm-specs/pull/125/commits/0912af8352fad807c9c70e00c0cd89c1e7532ded)